src/services/azblob/backend.rs

```diff
@@ -730,11 +812,172 @@ impl AzblobBackend {
 
         self.client.send_async(req).await
     }
+
+    async fn azblob_batch_del(
+        &self,
+        paths: impl Iterator<Item = &String>,
```

Comment:
Doesn't make sense to me. This API is not intended for public use and has very limited input types. Instead of using generics, I prefer to directly specify the type.